packages/cli/src/cli/config/action.import.ts

```diff
@@ -65,6 +66,11 @@ export class CommandImport extends CommandLineAction {
       parameterLongName: '--output',
       description: 'Output a markdown file with the config changes',
     });
+    this.target = this.defineStringParameter({
+      argumentName: 'TARGET',
+      parameterLongName: '--target',
+      description: 'import location',
```

Comment:
```suggestion
      description: 'Target config file to compare and replace',
```

Reply:
I wonder if target is the wrong word here.

Since its not replaced unless its dynamodb

Reply:
Yeah, `target` is for the future if we deprecate dyanmodb.

Reply:
I think we just remove the word replace for now.
```suggestion
      description: 'Target config file to compare',
```